tests/Oryx.Common.Tests/FileExtensionsTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Tests.Common;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.Oryx.Common.Extensions
+{
+    public class FileExtensionsTest : IClassFixture<TestTempDirTestFixture>
+    {
+        private TestTempDirTestFixture _testDir;
+        private string _testDirPath;
+
+        public FileExtensionsTest(TestTempDirTestFixture testFixture)
+        {
+            _testDir = testFixture;
+            _testDirPath = testFixture.RootDirPath;
+        }
+
+        [Fact]
+        public void SafeWriteAllText_Sanity()
```

Comment:
This is grouping multiple test scenarios in one method...It is better to have separate methods because a failure in one test blocks running other tests in this method (sure you can run the build again to find the next failure but that is not ideal) and also if we start looking at a test method's run history we would need to consider mixing multiple scenarios